src/main/scala/com/typesafe/conductr/sandbox/sbt/package.scala

```diff
@@ -0,0 +1,13 @@
+package com.typesafe.conductr.sandbox
+
+package object sbt {
+
+  implicit class TraversableOps[A](trav: Traversable[A]) {
+
+    def toMapWithKey[B](key: String)(f: Traversable[A] => B): Map[String, B] =
+      if (trav.isEmpty)
+        Map.empty[String, B]
+      else
+        Map(key -> f(trav))
+  }
+}
```

Comment:
Could `zipAll` be used instead? I'm skeptical of needing to add our own collection utilities...